osmosis/assetlist.json

```diff
@@ -14261,6 +14261,26 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/bOSMO.png"
         }
       ]
+    },
+    {
+      "description": "Memecoin for The International Brane Wave",
+      "denom_units": [
+        {
+          "denom": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+          "exponent": 0
+        },
+        {
+          "denom": "brnz",
+          "exponent": 0
+        }
+      ],
+      "base": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+      "name": "BRNZ",
+      "display": "brnz",
+      "symbol": "BRNZ",
+      "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/BRNZ.svg"
```

Comment:
Please provide an image with a square aspect ratio. (same width as height)

Reply:
Added new image, more coin-like, should be 2222x2222